platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/BasicConfig.java

```diff
@@ -251,4 +263,10 @@ public record BasicConfig(
         @ConfigProperty(value = "socketIpTos", defaultValue = "-1") int socketIpTos,
         @ConfigProperty(value = "maxIncomingSyncsInc", defaultValue = "1") int maxIncomingSyncsInc,
         @ConfigProperty(value = "maxOutgoingSyncs", defaultValue = "2") int maxOutgoingSyncs,
-        @ConfigProperty(value = "logPath", defaultValue = "log4j2.xml") Path logPath) {}
+        @ConfigProperty(value = "logPath", defaultValue = "log4j2.xml") Path logPath,
```

Comment:
we should really try to move away from using BasicConfig and have smaller, more specialized configs

Reply:
I split the sync config values into a new file `SyncConfig`